cdap-data-fabric-tests/src/test/java/co/cask/cdap/data/stream/service/ConcurrentStreamWriterTestBase.java

```diff
@@ -73,13 +74,13 @@
   @Test
   public void testConcurrentWrite() throws Exception {
     final String streamName = "testConcurrentWrite";
-    String accountId = "account";
+    String namespace = "namespace";
```

Comment:
I didn't want to pull up Id.Stream as a constant for the entire test, because the multiple test cases use different stream names. (I didn't want to make a functional difference in the test cases).